code-challenges/sort/challenges-03.js

```diff
@@ -179,15 +192,15 @@ xdescribe('Testing challenge 1', () => {
   });
 });
 
```

Comment:
dont forget to un-skip the tests when you submit, otherwise i cant grade if they pass or not